### DISCUSSION TITLE
Reinventando as Constelações do Zodíaco

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#392] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/reinventando-as-constelacoes-do-zodiaco/

**Comment by one-eyed-willy** [comment]
Interessante... Não sabia que o Aldebaran era brasileiro.

**Comment by brunoaugusto** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/reinventando-as-constelacoes-do-zodiaco/#comment_1087793890>):

> Interessante... Não sabia que o Aldebaran era brasileiro.

Tem [um mapa](<https://i.postimg.cc/sgYj4ctb/mapa-jogodx4.jpg>) no verso do tabuleiro de um jogo lançado pela Bandai no final dos anos 80 onde mostra o local de nascimento/treinamento de alguns Cavaleiros e, pela posição do indicador, ele nasceu/treinou na região do Pará. Não sei se ele realmente nasceu lá porque eu não sei ler japonês pra ler o mapa.

**Comment by tali-zorah** [comment]
*Keelah!* Nunca achei que fosse gostar tanto de uma série assim até assistir pela primeira vez

**Comment by brunoaugusto** [comment]
> [Tali'Zorah nar Rayya](</perfil/?user=tali-zorah>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/reinventando-as-constelacoes-do-zodiaco/#comment_1087812810>):

> *Keelah!* Nunca achei que fosse gostar tanto de uma série assim até assistir pela primeira vez

Foi você quem desenhou as constelações da imagem?

**Comment by tali-zorah** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/reinventando-as-constelacoes-do-zodiaco/#comment_1087814090>):

> Foi você quem desenhou as constelações da imagem?

Só montei uma versão traduzida com uma foto que eu achei porque a qualidade da original estava ruim
